action-api/src/test/resources/ActionFileTest.json

```diff
@@ -189,5 +189,15 @@
     "elementId" : "transformerId25",
     "side" : "THREE",
     "open" : true
+  }, {
+    "type" : "AREA_INTERCHANGE_TARGET_UPDATE_ACTION",
+    "id" : "id99",
+    "areaId" : "AreaA",
+    "target" : 101.0
+  }, {
+    "type" : "AREA_INTERCHANGE_TARGET_UPDATE_ACTION",
+    "id" : "idDisabledTarget",
+    "areaId" : "AreaA",
+    "target" : NaN
```

Comment:
I think it would be better not to write the field. `NaN` is not standard in JSON / `NaN` support is JSON parser dependent.

Reply:
Agreed I setup NaN as the default value in the builder and simply do not serialize it when the value is NaN.